examples/js/examples.js

```diff
@@ -378,32 +380,28 @@ function addOpenInCodePenForm (exampleIndex, exampleHeaderId, exampleCodeId, exa
               totalFetchedFiles++;
           }
           else {
-              hideButton(buttonId, "Could not load resource: " + href);
+              console.log("Not showing 'Open in Codepen' button. Could not load resource: " + href);
```

Comment:
Does it make sense to warn/error for these?
```suggestion
              console.warn("Not showing 'Open in Codepen' button. Could not load resource: " + href);
```

Reply:
makes sense! pushed commit for switch.